Pediatrics.csv

```diff
@@ -256,23 +256,23 @@ ID,Study,Reference,Grouping,Analyte,t0,tend,t Unit,AUC Avg,AUC AvgUnit,AUC AvgTy
+5213,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID4-DDI 3y5m,Digoxin,0,24,h,,,,,,,,,,,,,,,3.979663212,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5214,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID5-2w,Digoxin,0,24,h,,,,,,,,,,,,,,,3.906933333,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5215,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID5-DDI 4w,Digoxin,0,24,h,,,,,,,,,,,,,,,3.366,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5216,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID6-6m,Digoxin,0,24,h,,,,,,,,,,,,,,,3.851851852,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5217,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID6-DDI 8m,Digoxin,0,24,h,,,,,,,,,,,,,,,2.411111111,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5218,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID7-8m,Digoxin,0,24,h,,,,,,,,,,,,,,,5.93614304,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5219,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID7-DDI 9m,Digoxin,0,24,h,,,,,,,,,,,,,,,2.610381862,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5220,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 7m,Digoxin,0,24,h,,,,,,,,,,,,,,,8.508641975,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5221,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 8m,Digoxin,0,24,h,,,,,,,,,,,,,,,4.537931034,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5222,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,neonates,Digoxin,0,24,h,,,,,,,,,,,,,,,3,mL/min/kg,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations
+5223,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,infants (moderate cl),Digoxin,0,24,h,,,,,,,,,,,,,,,4.5,mL/min/kg,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations

```

Comment:
The values for Nyberg in `ml/min/kg `are the same as before in `mL/min/1.73m2` ?

Reply:
correct, it was a copy error.

Reply:
ok, after this was fixed, we can merge